sdk/core/core-amqp/src/retry.ts

```diff
@@ -111,38 +111,41 @@ async function checkNetworkConnection(host: string): Promise<boolean> {
 }
 
 /**
- * It will attempt to linearly retry an operation specified number of times with a specified
- * delay in between each retry. The retries will only happen if the error is retryable.
+ * By default an operation is attempted only once.
```

Comment:
We shouldnt use the phrase "By default" here, because by default we retry thrice :)
How about, `Every operation is attempted atleast once`